parametric/test_headers_b3multi.py

```diff
@@ -53,7 +52,7 @@ def test_headers_b3multi_extract_valid(test_agent, test_library):
 
 
 @enable_b3multi()
-@pytest.mark.skip_library("golang", "not implemented")
+@pytest.mark.skip_library("golang", "BUG: the parent_id will be 0 in any span that didn't have a parent context ; if extract fails - the parent context is nil")
```

Comment:
Let's link to the GH issue for this so we can track it. Here and throughout. If it's something that's intended behavior that we don't intend to fix, then let's make sure to indicate that clearly.

Reply:
This was mentioned in a different pr for node, and will be fixed later